bin/audit-svg-colors.js

```diff
@@ -16,34 +16,81 @@ const { readFileSync } = require( 'fs' );
 const path = require( 'path' );
 const PALETTE = require( '@automattic/color-studio' );
 const chroma = require( 'chroma-js' );
-const _ = require( 'lodash' );
 
 /**
  * Palette color subsets
  */
 
+/**
+ * pickBy function is a replacement for the Lodash ._pickby
+ *
+ * @param {*} obj a palette of colors
+ * @param {*} fn This function returns truthy or falsy value to filter the colors
+ * @returns objext with filtered color values
+ * Used Object.keys() and Array.prototype.filter()to remove the keys for which fn returns a falsy value.
+ * Used Array.prototype.reduce() to convert the filtered keys back to an object with the corresponding key-value pairs.
+ */
+const pickBy = ( obj, fn ) =>
+	Object.keys( obj )
+		.filter( ( key ) => fn( obj[ key ], key ) )
+		.reduce( ( acc, key ) => ( ( acc[ key ] = obj[ key ] ), acc ), {} );
+
+/**
+ * Native Sort - replacement of ._sortBy Lodash function
+ *
+ * @param {*} key
+ * @returns Sorted array. The native sort modifies the array in place. So we use `.concat()` to
+ *  copy the array, then sort.
+ */
+const sortBy = ( key ) => {
+	return ( a, b ) => {
+		if ( a[ key ] > b[ key ] ) {
+			return 1;
+		} else if ( b[ key ] > a[ key ] ) {
+			return -1;
+		}
+		return 0;
+	};
+};
+
 // The subset of palette colors allowed in illustrations
-const PALETTE_ILLUSTRATION_COLORS = _.pickBy( PALETTE.colors, ( colorValue, colorName ) => {
+const PALETTE_ILLUSTRATION_COLORS = () => {
```

Comment:
@yashitamittal11, as you can see, previously we had a function call (`_.pickBy( ... )`). 
The newer version (instead of getting a value of function call) assign an arrow function to a constant.
It seems, we need to call something to put the required value into the constant.